notifications/tasks.py

```diff
@@ -0,0 +1,18 @@
+from celery import shared_task
+import requests
+from django.conf import settings
+from habits.models import Habit
+from datetime import datetime
+
+
+@shared_task
+def send_telegram_reminder():
+    now = datetime.now().time()
+    habits = Habit.objects.filter(time__hour=now.hour, time__minute=now.minute)
```

Comment:
Так как далее ты достаешь из каждой привычки пользователя, то информацию по нему проще сразу подтянуть из БД, причем тебя не интересуют пользователи, у которых не проставлен `chat_id` - им не нужно ничего отправлять
```suggestion
    habits = Habit.objects.select_related('user').filter(time__hour=now.hour, time__minute=now.minute).exclude(user__telegram_chat_id__isnull=True)
```